README.md

```diff
@@ -218,6 +218,23 @@ class ProblemNameTest < Minitest::Test
   end
 end
 ```
+### Workload philosophy.
+
+Within the test workload we want to prioritize educational value over expert comprehension and want to make sure that things are clear to people who may not be familiar with Minitest and even Ruby. 
+
+Therefore a multi line workload is preferred. 
```

Comment:
I would just omit this line and put the (revised paragraph) below the example here.